ios/browser/providers/BUILD.gn

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2021 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# This is a variant of //ios/chrome/browser/providers:chromium_providers which
+# does not include bundling the MaterialComponents framework
+group("brave_providers") {
+  deps = [
+    # The target providing the ChromeBrowserProvider factory.
+    "//ios/chrome/browser/providers:chromium_provider_factory",
+
+    # The individual API implementations.
+    "//ios/chrome/browser/providers/modals:chromium_modals",
```

Comment:
if these aren't needed we should leave them out

Reply:
So I made a build with them removed and it failed, so unfortunately both of these are needed